ctod.dd

```diff
@@ -208,14 +208,14 @@ $(H3 <a name="nans">Dealing with NANs in floating point compares</a>)
 $(H4 The C Way)
 
        C doesn't define what happens if an operand to a compare
-       is NAN, and few C compilers check for it (the Digital Mars
-       C compiler is an exception, DM's compilers do check for NAN operands).
+       is `NAN`, and few C compilers check for it (the Digital Mars
+       C compiler is an exception, DM's compilers do check for `NAN` operands).
 
 $(CCODE
 #include &lt;math.h&gt;
 
 if (isnan(x) || isnan(y))
-    result = FALSE;
+    result = false;
```

Comment:
Since C99 bool type is defined in the language and `true` and `false` are keyword.